src/AppBundle/Resources/views/Admin/_edit.html.twig

```diff
@@ -31,6 +31,12 @@ transDomain: 'admin'
         <li>Client name: {{ firstClient.firstname }}</li>
         <li>Client lastname: {{ firstClient.lastname }}</li>
         <li>Case number: {{ firstClient.caseNumber }}</li>
+        {% if firstClient.odr %}
```

Comment:
@sheanie very minor thing. You can use "and" with twig if you want to collapse the two nested "if"